database/insert_data.sh

```diff
@@ -1,9 +1,5 @@
 #!/bin/bash
 
-#copy folder with snapshot_basic for inital Application manager setup
-mkdir ./database/_snapshots
-cp -rf ./database/core_templates ./database/_snapshots
```

Comment:
This is all done as part of `createDefaultDataFolders()` which runs on server startup as well as snapshot take/use scripts